src/components/dataspaceDashboard/dataspaceEntities/DataspaceEntities.tsx

```diff
@@ -31,22 +32,39 @@ const DataspaceEntities = ({ dataspaceId }: DeepReadonly<{ dataspaceId: string }
   return (
     <div className={classNames('okp4-dashboard-dataspace-content', theme)}>
       {entities.map(
-        (entity: DeepReadonly<DataverseEntity>): JSX.Element => (
-          <Card
-            footer={
-              <div className="okp4-dataspace-card-footer">
-                <Typography>{entity.name}</Typography>
-              </div>
-            }
-            header={
-              <div className="okp4-dataspace-card-header">
-                <Typography>{entity.type}</Typography>
-              </div>
-            }
-            key={entity.id}
-            size="small"
-          />
-        )
+        ({ id, name, type }: DeepReadonly<DataverseEntity>): JSX.Element =>
+          type === 'dataset' ? (
+            <Link href={`/dataverse/explore/dataspace/${dataspaceId}/dataset/${id}`} key={id}>
+              <Card
+                footer={
+                  <div className="okp4-dataspace-card-footer">
+                    <Typography>{name}</Typography>
+                  </div>
+                }
+                header={
+                  <div className="okp4-dataspace-card-header">
+                    <Typography>{type}</Typography>
+                  </div>
+                }
+                size="small"
+              />
+            </Link>
+          ) : (
+            <Card
```

Comment:
You can refactor this lines of code as you repeat yourself twice with the `Card` component.